### DISCUSSION TITLE
about log level

### DISCUSSION THREAD

**Discussion by lopo12123** [farm-fe/farm#916] [General]
![image](https://github.com/farm-fe/farm/assets/59609929/9b09357e-4340-4324-98f4-98c9744e9fe5)
![image](https://github.com/farm-fe/farm/assets/59609929/b64588a5-8270-42ca-a6c0-20ed6895ec8a)

Maybe it would be more friendly to use the verbose level (with `console.debug`) for these informations (to avoid being mixed with some of your own log information during development and to be easily filtered

Just for discussion here, there is nothing wrong with the current look :)

**Comment by wre232114** [comment]
Thanks, that's a good suggestion. we will use `console.debug` next version.

**Reply by wre232114** [reply]
supported since `@farmfe/core@0.16.1`
